packages/fast-components-react-base/src/media/DOCS.MD

```diff
@@ -0,0 +1,11 @@
+## Best practices
+Use *image* to break up text-heavy pages, to illustrate products for sale, or to improve storytelling on a page. The use of *image* should always support the page’s story and, when possible, contribute to that story. Size the *image* to complement the rest of the page content. Use the **picture** option to employ a selection of image versions for different viewport sizes.
+
+### Usage guidance
+Use the [Lifestyle](/getting-started/designing/imagery-lifestyle) and [Product](/getting-started/designing/imagery-product) imagery guidance when choosing your images. Pay attention to guidance for improper usage at the bottom of the Lifestyle and Product imagery pages. Use HTML text instead of putting graphic text in images.
+
+### Style guidance

```

Comment:
FYI @awentzel - Below is a perfect example of something specific to MSFT.

Reply:
See above for same answer